### DISCUSSION TITLE
Doc specifies you can use celery with Fargate/serverless even though you use celery.

### DISCUSSION THREAD

**Discussion by Andrew-Chen-Wang** [intelowlproject/IntelOwl#96] [General]
In docs/source/Installation.md, you say you can deploy serverless, even though you use celery. For an application like this, I cannot warn enough against this. You can view an SO question which explains why not here: https://stackoverflow.com/questions/53218715/operating-the-celery-worker-in-the-ecs-fargate

A previous talk with someone else mentioned pretty much the same points when using AWS lambda. I suggest you try to do AWS ECS or EC2 with CodeDeploy from your repository.

**Comment by mlodic** [comment]
As said here #95, the AWS support for serverless deployment is a "Future" idea.

We will leverage your tips when we will get back into this! Thank you!
